### PR TITLE
isCurrentSmove logic added upon adding new smove. Updated profile route, major styling

### PR DESCRIPTION
**Pull request by danitian**

### CODE REVIEW

**Comment by danitian** [comment]
- fixed isCurrentSmove logic to make all other smoves not the current smove upon new smove creation
- updated profiles route so that other components display without the profile component when clicked
- major styling updates

**Comment by danitian** [comment]
- also handles the case where user has no smoves

File: client/components/Header.jsx [hunk 1]
```diff
@@ -5,21 +5,21 @@ import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
 import Nav from 'react-bootstrap/Nav';
 import Navbar from 'react-bootstrap/Navbar'
+import Logout from './login/Logout'
 
 const Header = () => {
   return (
     <React.Fragment>
       <Navbar bg="dark" variant="dark">
```
Comment (grsc0529) [inline client/components/Header.jsx:13]: I will fix these links on the next PR

**Review by greta-schock — APPROVED** [review]
lgtm

**Review by mingwencode — APPROVED** [review]

**Review by johnkwak08 — COMMENTED** [review]
LGTM!

**Pull request merged**